rust/src/accuracy/polars/mod.rs

```diff
@@ -104,7 +106,16 @@ fn logical_plan_utility(
         DslPlan::Select { expr: exprs, .. } | DslPlan::GroupBy { aggs: exprs, .. } => {
             let rows = exprs
                 .iter()
-                .map(|e| expr_utility(&e, alpha, threshold.clone()))
+                .map(|e| {
+                    let name = e.clone().meta().output_name()?.to_string();
```

Comment:
This is a little extra. It ensures that the column name matches, even when summarizing quantile statistics.